ironfish-cli/.yarnrc

```diff
@@ -0,0 +1 @@
+--silent true
```

Comment:
This is the tricky thing I was trying to  point out. If you do this, it breaks all the logs for developers using yarn commands for install, build, etc because nothing is printed. Honestly, I think if you're using from source you have to be prepared to use engineering tooling and know how it works and be okay with these errors.

We're going to release NPM builds soon so 90% of users won't be running from source anymore and therefore should not be running yarn.